docs/ref/forms/fields.txt

```diff
@@ -258,6 +258,36 @@ fields. We've specified ``auto_id=False`` to simplify the output::
     <p>Sender: <input type="email" name="sender" required> A valid email address, please.</p>
     <p>Cc myself: <input type="checkbox" name="cc_myself"></p>
 
+When a field has help text and :attr:`~django.forms.BoundField.id_for_label`
+returns a value `help_text` will be associated with the input using
+``aria-describedby``::
+
+    >>> from django import forms
+    >>> class UserForm(forms.Form):
+    ...     username = forms.CharField(max_length=10, help_text='e.g., user@example.com')
+    >>> f = UserForm()
+    >>> print(f.as_p())
```

Comment:
Can we use the `as_div()` style. It didn't exist when I originally prepared this patch so making the change now will be one less for someone (me?) to change later.